vhdbuilder/packer/components.json

```diff
@@ -180,7 +180,7 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/tigera/operator:*",
       "versions": [
-        "v1.13.5"
+        "v1.17.1"
```

Comment:
please append new versions, and keep the old versions until the weekly OverlayMgr release for the new image is rolled out to all regions.

Reply:
We're currently going from v1.15.1 -> v1.17.1, so I think we forgot to update that previously and that's why we have v1.13.5 here. I could add in v1.15.1 as well if it would make sense to do so.